packages/syntax-core/src/LinkButton/LinkButton.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { type StoryObj, type Meta } from "@storybook/react";
+import LinkButton from "./LinkButton";
+import FavoriteBorder from "@mui/icons-material/FavoriteBorder";
+
+export default {
+  title: "Components/LinkButton",
+  component: LinkButton,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/Cambly-Design-System-(Draft)?node-id=994%3A2346",
+    },
+  },
+  argTypes: {
+    color: {
+      text: {
+        control: { type: "text" },
+      },
+      href: {
+        control: { type: "text" },
+      },
+      target: {
```

Comment:
@somethiiing Can we make `target` more restrictive? Just like we defined our types: `target?: "_blank" | "_self" | "_parent" | "_top";`